### ISSUE TITLE
java.lang.NumberFormatException: For input string: "0-RC2"

### ISSUE THREAD

**Issue by sergiuszkierat** [playframework/playframework#9391]
### Play Version

2.7.2

### Sbt Version

1.3.0-RC2

### API

Scala

### Operating System

Ubuntu 18.04.2 LTS

### JDK

openjdk version "1.8.0_212"
OpenJDK Runtime Environment (build 1.8.0_212-8u212-b03-0ubuntu1.18.04.1-b03)
OpenJDK 64-Bit Server VM (build 25.212-b03, mixed mode)

### Expected Behavior

1. git clone git@github.com:sergiuszkierat/play-samples.git
2. cd play-samples/play-scala-starter-example/
3. sbt
4. run
5. no error in the console

### Actual Behavior

1-4 as above
5. an error is being raised
```
[error] java.lang.NumberFormatException: For input string: "0-RC2"
[error]         at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
[error]         at java.lang.Integer.parseInt(Integer.java:580)
[error]         at java.lang.Integer.parseInt(Integer.java:615)
[error]         at scala.collection.immutable.StringLike.toInt(StringLike.scala:304)
[error]         at scala.collection.immutable.StringLike.toInt$(StringLike.scala:304)
[error]         at scala.collection.immutable.StringOps.toInt(StringOps.scala:33)
[error]         at play.sbt.run.PlayRunCompat.$anonfun$watchContinuously$7(PlayRunCompat.scala:63)
[error]         at play.sbt.run.PlayRunCompat.$anonfun$watchContinuously$7$adapted(PlayRunCompat.scala:63)
[error]         at scala.collection.TraversableLike.$anonfun$map$1(TraversableLike.scala:237)
[error]         at scala.collection.IndexedSeqOptimized.foreach(IndexedSeqOptimized.scala:36)
[error]         at scala.collection.IndexedSeqOptimized.foreach$(IndexedSeqOptimized.scala:33)
[error]         at scala.collection.mutable.ArrayOps$ofRef.foreach(ArrayOps.scala:198)
[error]         at scala.collection.TraversableLike.map(TraversableLike.scala:237)
[error]         at scala.collection.TraversableLike.map$(TraversableLike.scala:230)
[error]         at scala.collection.mutable.ArrayOps$ofRef.map(ArrayOps.scala:198)
[error]         at play.sbt.run.PlayRunCompat.watchContinuously(PlayRunCompat.scala:63)
[error]         at play.sbt.run.PlayRunCompat.watchContinuously$(PlayRunCompat.scala:38)
[error]         at play.sbt.run.PlayRun$.watchContinuously(PlayRun.scala:34)
[error]         at play.sbt.run.PlayRun$.$anonfun$playRunTask$3(PlayRun.scala:113)
[error]         at play.sbt.run.PlayRun$.$anonfun$playRunTask$3$adapted(PlayRun.scala:68)
[error]         at scala.Function1.$anonfun$compose$1(Function1.scala:49)
[error] (Compile / run) java.lang.NumberFormatException: For input string: "0-RC2"
[error] Total time: 3 s, completed Jun 13, 2019 7:10:31 PM
```

**Comment by mkurz** [comment]
Fixed by #9369 already. Will land in 2.7.2.

**Comment by sergiuszkierat** [comment]
Thanks @mkurz
